ir/json_generator.h

```diff
@@ -135,7 +135,10 @@ class JSONGenerator {
                 break;
             case OBJ_END:
                 break;
-            default:
```

Comment:
I wonder, should the `default:` still exist in case new enum value is added or somehow we get value which is not one of the enumerated once?

Reply:
Well, adding a new switch value will trigger the `-Wswitch-case` warning again, regardless of whether default is present or not.  Since getting a value that is not a valid tag value (from memory corruption or uninitialized data) would only be possible with undefined behavior, it is not clear what adding default would actually gain.